Add tools deps usage notes to README

lread opened the pull request:
This project depends on native dep org.graalvm.nativeimage/svm. 

While technically correct, tools deps does not support native deps and fails for this native dep.

Since the native dep is provided by the Graal installation anyway, it can be safely excluded for tools deps.

lread committed:
Add tools deps usage notes to README

This project depends on native dep org.graalvm.nativeimage/svm. 

While technically correct, tools deps does not support native deps and fails for this native dep.

Since the native dep is provided by the Graal installation anyway, it can be safely excluded for tools deps.

borkdude commented:
Can you add the comment you made in this issue to the README as well? That seems pretty informative.

lread committed:
Refine README Usage Section

Add in rationale for native dep exclusion for tools deps.
With this extra text, for clarity, reorganized usage into subsections.

lread commented:
Ya, good idea, done.

borkdude commented:
Thanks!
